moj-node/server/repositories/analytics.js

```diff
@@ -12,7 +12,7 @@ function analyticsRepository(httpClient) {
       el: label,
     };
 
-    if (value !== undefined) {
+    if (value) {
```

Comment:
Can you change this to match the branch as this doesn't work, needs to check for undefined :)

Reply:
Removed the commit